src/services/enterpriseService.ts

```
@@ -0,0 +1,312 @@
+  recipient: string,
+  amountAcbu: string,
+  reference?: string,
+  rowKey?: string,
+): string {
+  return crypto
+    .createHash("sha256")
+    .update([organizationId, recipient, amountAcbu, reference ?? "", rowKey ?? ""].join("|"))
+    .digest("hex");
+}
+
+function parseCsvBuffer(buffer: Buffer): { headers: string[]; rows: Record<string, string>[] } {
```

ESLint

[error] Replace `·headers:·string[];·rows:·Record<string,·string>[]·` with `⏎··headers:·string[];⏎··rows:·Record<string,·string>[];⏎`

(prettier/prettier)